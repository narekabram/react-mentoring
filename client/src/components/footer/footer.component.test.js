import React from 'react';
import { shallow } from 'enzyme';
import Footer from './footer';

describe('Footer', () => {
    it('Adds Footer snapshot testing', () => {
        const component = shallow(<Footer/>);

        expect(component).toMatchSnapshot();
    });
});
