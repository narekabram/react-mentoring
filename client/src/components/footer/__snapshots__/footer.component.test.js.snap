// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Adds Footer snapshot testing 1`] = `ShallowWrapper {}`;
